medicines/doc-index-updater/src/create_manager/mod.rs

```diff
@@ -195,13 +196,12 @@ mod test {
     #[test]
     fn test_an_unknown_error_does_not_remove_create_message() {
         let state_manager = TestJobStatusClient {};
-        let mut removeable = TestRemoveable { is_removed: false };
-        let message = given_we_have_a_create_message();
+        let removeable_message = &mut given_we_have_a_create_message();
```

Comment:
I think this is better as `let mut removeable_message = given…`

Reply:
Yes, seems nicer, done